chromatic/tests/test_shift.py

```diff
@@ -0,0 +1,40 @@
+from ..rainbows import *
+from .setup_tests import *
+
+
+def test_shift():
```

Comment:
I added this test function to make sure that the code is doing what we're expecting. This will run automatically any time that you run `pytest` from within the main repository directory (see #7 in [this list](https://zkbt.github.io/chromatic/designing/#how-do-we-add-new-abilities-to-rainbow-objects)). But, you can also run it from within a notebook just by calling:

```python
from chromatic.tests import *
test_shift()
```

It should run without errors. Does it? 

I really like including tests with as much new code as possible, and this function is a particularly good one to test because there are straightforward quantitative checks we can apply (= "if we redshift the spectrum do its wavelengths get longer?").